relay-server/src/metrics_extraction/spans/mod.rs

```diff
@@ -186,11 +187,17 @@ pub(crate) fn extract_span_metrics(
                 );
 
                 if let Some(scrubbed_desc) = sanitized_description {
-                    span_tags.insert(SpanTagKey::Description, scrubbed_desc.to_owned());
+                    // Truncating the span description's tag value is, for now,
+                    // a temporary solution to not get large descriptions dropped. The
```

Comment:
what would the permanent solution be ?

Reply:
At this point, unknown. We will need more product iterations to see what the best action is; maybe we want to increase the limit of the values instead.